darts/tests/models/forecasting/test_torch_forecasting_model.py

```diff
@@ -78,6 +79,20 @@
 ]
 
 
+class NumsCalled(Metric):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.add_state("preds", default=[], dist_reduce_fx="cat")
+
+    def update(self, preds, target) -> None:
+        if preds.shape != target.shape:
+            raise ValueError("preds and target must have the same shape")
```

Comment:
```suggestion
            raise ValueError(f"preds and target must have the same shape, but got {preds.shape} for preds and {target.shape} for target.")
```

Reply:
Thanks, updated. Note that this is just a test class, this error isn't raised in the training loop itself